booking/src/main/java/org/dcsa/conformance/standards/booking/action/StateChangingBookingAction.java

```diff
@@ -1,7 +1,21 @@
 package org.dcsa.conformance.standards.booking.action;
 
+import org.dcsa.conformance.core.check.ActionCheck;
+import org.dcsa.conformance.core.check.ApiHeaderCheck;
+import org.dcsa.conformance.core.check.ConformanceCheck;
+import org.dcsa.conformance.core.check.HttpMethodCheck;
+import org.dcsa.conformance.core.check.JsonSchemaCheck;
+import org.dcsa.conformance.core.check.JsonSchemaValidator;
+import org.dcsa.conformance.core.check.ResponseStatusCheck;
+import org.dcsa.conformance.core.check.UrlPathCheck;
 import org.dcsa.conformance.core.traffic.ConformanceExchange;
+import org.dcsa.conformance.core.traffic.HttpMessageType;
+import org.dcsa.conformance.standards.booking.checks.BookingChecks;
 import org.dcsa.conformance.standards.booking.checks.ScenarioType;
+import org.dcsa.conformance.standards.booking.party.BookingRole;
+
+import javax.swing.*;
```

Comment:
I guess this import is wrong. We're not using Java Swing elements..

Reply:
I dont know when this got imported.  will remove the unused imports